test/conftest.py

```diff
@@ -8,7 +8,7 @@
 from dsp_tools.utils.logger_config import logger_config
 
 
-@pytest.fixture()
+@pytest.fixture
```

Comment:
Rule change in ruff 0.6.0. As per the release notes (https://astral.sh/blog/ruff-v0.6.0#default-behavior-changed-for-flake8-pytest-style-rules), the new rule adheres more closely to the style recommendations made by the official pytest project